buildSrc/src/main/java/com/mertceyhan/buildsrc/Dependencies.kt

```diff
@@ -55,5 +55,6 @@ object Dependencies {
     const val roomKtx = "androidx.room:room-ktx:${Versions.roomVersion}"
     const val roomCompiler = "androidx.room:room-compiler:${Versions.roomVersion}"
     const val glance = "androidx.glance:glance-appwidget:${Versions.glance}"
+    const val accompanist = "com.google.accompanist:accompanist-placeholder:${Versions.accompanist}"
```

Comment:
I think `placeholder` would be a better name. `accompanist` is the name of the whole library but we use just place older, right?

`const val placeholder = "com.google.accompanist:accompanist-placeholder:${Versions.placeholderVersion}"`